src/main/java/edu/ucla/library/prl/harvester/services/HarvestScheduleStoreServiceImpl.java

```diff
@@ -83,19 +85,25 @@ public class HarvestScheduleStoreServiceImpl implements HarvestScheduleStoreServ
     /**
      * The select-one query for jobs.
      */
-    private static final String GET_JOB = "SELECT * FROM public.harvestjobs WHERE id = $1";
+    private static final String GET_JOB =
+            "SELECT institutionId AS \"institutionId\", repositoryBaseUrl AS \"repositoryBaseUrl\", " +
+                    "metadataPrefix AS \"metadataPrefix\", sets, lastSuccessfulRun AS \"lastSuccessfulRun\", " +
+                    "scheduleCronExpression AS \"scheduleCronExpression\" FROM public.harvestjobs WHERE id = $1";
 
     /**
      * The insert query for jobs.
      */
     private static final String ADD_JOB =
-            "INSERT INTO public.harvestjobs(institutionID, repositoryBaseURL, metadataPrefix, sets," +
+            "INSERT INTO public.harvestjobs(institutionId, repositoryBaseUrl, metadataPrefix, sets," +
                     " lastSuccessfulRun, scheduleCronExpression) VALUES($1, $2, $3, $4, $5, $6) RETURNING id";
 
     /**
-     * The select query for all jobs.
+     * The select query for all jobs. N.b. awkward format to satisfy checkstyle rules
```

Comment:
Is the checkstyle rule here line length or something else?

Reply:
The GET_JOB and LIST__JOBS queries originally had same layout for SELECT sections, and checkstyle was complaining about duplicate string literals.

Reply:
I'm not sure I follow. One was `"SELECT * FROM public.harvestjobs WHERE id = $1"` and the other `"SELECT * FROM public.harvestjobs ORDER BY institutionID"` which are different string literals(?)

Unrelated then, as a general strategy instead of concatenating string parts into a single string with `+`s, heredocs are supported in Java now:

```
private static final String LIST_JOBS = """
      SELECT institutionId AS \"institutionId\", repositoryBaseUrl AS \"repositoryBaseUrl\",
      metadataPrefix AS \"metadataPrefix\", sets, lastSuccessfulRun AS \"lastSuccessfulRun\",
      scheduleCronExpression AS \"scheduleCronExpression\" FROM public.harvestjobs ORDER BY \"institutionId\"
    """;
```

Reply:
Oh do you mean in the new form there were string parts that were shared across the queries. I think I see that now. Heredocs would also solve that.

Reply:
As an aside, I'm kind of curious whether we'd need to use the backslash escaping of quotes with the text block format(?) The JEP SQL example shows just the standalone quotes (no escaping).

Reply:
Switched to heredocs for long query strings.